README.md

```diff
@@ -152,18 +158,69 @@ To create a `task definition`, just `POST` to the endpoint:
 {}
 ```
 
-## Development
+## Secrets
+
+`Secrets` store binary or string data in AWS secrets manager. By default, secrets are encrypted (in AWS) by the `defaultKmsKeyId` given for each `account`.
+
+### Create a secret
 
-    - Install Go v1.11 or newer
-    - Enable Go modules: `export GO111MODULE=on`
-    - Create a config file with your account parameters:
+POST `/v1/s3/{account}/secrets`
+
+#### Request
+
+```json
+{
+    "Name": "sshhhhh",
+    "SecretString": "abc123"
+}
+```
+
+#### Response
+
+```json
+{
+    "ARN": "arn:aws:secretsmanager:us-east-1:012345678901:secret:sshhhhh-Z8CxfW",
+    "Name": "sshhhhh",
+    "VersionId": "592CEFAE-7B74-4A22-B1C9-55F958531579"
+}
 ```
-$ cp -p config/config.example.json config/config.json
-# edit config.json and update the parameters
+
+| Response Code                 | Definition                      |
+| ----------------------------- | --------------------------------|
+| **200 OK**                    | okay                            |
+| **400 Bad Request**           | badly formed request            |
+| **500 Internal Server Error** | a server error occurred         |
+
+### List secrets
+
+Listing secrets is limited to the secrets that belong to the *org*.
```

Comment:
It's unclear what this `org` is and how it's configured, maybe we should elaborate.